Trackers/iou_pred_tracker/iou_pred.py

```diff
@@ -6,10 +6,16 @@
 class iou_pred_track(Track):
     def __init__(self, id, bbox, hits, miss):
         self.history=[]
+        self.hiscount = 5
         Track.__init__(self, id, bbox, hits, miss)
 
     def iou_predict(self):
-       
+        if(self.hits < self.hiscount):
```

Comment:
I think it would be better to check `len(self.history)` rather then `self.hits` since it will create less bugs in case of modification of logic in future

Reply:
Done